[2024-04-23T00:00:48.766+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-04-22T00:00:00+00:00 [queued]>
[2024-04-23T00:00:48.772+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-04-22T00:00:00+00:00 [queued]>
[2024-04-23T00:00:48.773+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-04-23T00:00:48.914+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): normalize_numerical_features> on 2024-04-22 00:00:00+00:00
[2024-04-23T00:00:48.919+0000] {standard_task_runner.py:60} INFO - Started process 45037 to run task
[2024-04-23T00:00:48.921+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'process_etl_heart_data', 'normalize_numerical_features', 'scheduled__2024-04-22T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/etl_process.py', '--cfg-path', '/tmp/tmpg505l1ic']
[2024-04-23T00:00:48.923+0000] {standard_task_runner.py:88} INFO - Job 28: Subtask normalize_numerical_features
[2024-04-23T00:00:49.154+0000] {task_command.py:423} INFO - Running <TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-04-22T00:00:00+00:00 [running]> on host d1bc6a951fa1
[2024-04-23T00:00:49.506+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Facundo Adrian Lucianna' AIRFLOW_CTX_DAG_ID='process_etl_heart_data' AIRFLOW_CTX_TASK_ID='normalize_numerical_features' AIRFLOW_CTX_EXECUTION_DATE='2024-04-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-22T00:00:00+00:00'
[2024-04-23T00:00:49.508+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvm8jc_3g0 --system-site-packages --python=python
[2024-04-23T00:00:49.514+0000] {process_utils.py:186} INFO - Output:
[2024-04-23T00:00:49.938+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 153ms
[2024-04-23T00:00:49.939+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venvm8jc_3g0, clear=False, no_vcs_ignore=False, global=True)
[2024-04-23T00:00:49.939+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-04-23T00:00:49.940+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.3.1, setuptools==69.0.2, wheel==0.42.0
[2024-04-23T00:00:49.940+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-04-23T00:00:49.973+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvm8jc_3g0/bin/pip install -r /tmp/venvm8jc_3g0/requirements.txt
[2024-04-23T00:00:49.980+0000] {process_utils.py:186} INFO - Output:
[2024-04-23T00:00:51.152+0000] {process_utils.py:190} INFO - Collecting awswrangler==3.6.0 (from -r /tmp/venvm8jc_3g0/requirements.txt (line 1))
[2024-04-23T00:00:51.153+0000] {process_utils.py:190} INFO -   Using cached awswrangler-3.6.0-py3-none-any.whl.metadata (23 kB)
[2024-04-23T00:00:51.317+0000] {process_utils.py:190} INFO - Collecting mlflow==2.10.2 (from -r /tmp/venvm8jc_3g0/requirements.txt (line 2))
[2024-04-23T00:00:51.318+0000] {process_utils.py:190} INFO -   Using cached mlflow-2.10.2-py3-none-any.whl.metadata (13 kB)
[2024-04-23T00:00:51.320+0000] {process_utils.py:190} INFO - Requirement already satisfied: scikit-learn==1.3.2 in /home/***/.local/lib/python3.8/site-packages (from -r /tmp/venvm8jc_3g0/requirements.txt (line 3)) (1.3.2)
[2024-04-23T00:00:51.351+0000] {process_utils.py:190} INFO - Requirement already satisfied: boto3<2.0.0,>=1.20.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (1.34.88)
[2024-04-23T00:00:51.351+0000] {process_utils.py:190} INFO - Requirement already satisfied: botocore<2.0.0,>=1.23.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (1.34.88)
[2024-04-23T00:00:51.352+0000] {process_utils.py:190} INFO - Requirement already satisfied: numpy<2.0,>=1.18 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (1.24.4)
[2024-04-23T00:00:51.353+0000] {process_utils.py:190} INFO - Requirement already satisfied: packaging<24.0,>=21.1 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (23.2)
[2024-04-23T00:00:51.353+0000] {process_utils.py:190} INFO - Requirement already satisfied: pandas<2.1.0,>=1.2.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (2.0.3)
[2024-04-23T00:00:51.354+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyarrow>=7.0.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (14.0.2)
[2024-04-23T00:00:51.354+0000] {process_utils.py:190} INFO - Requirement already satisfied: typing-extensions<5.0.0,>=4.4.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (4.9.0)
[2024-04-23T00:00:51.390+0000] {process_utils.py:190} INFO - Requirement already satisfied: click<9,>=7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (8.1.7)
[2024-04-23T00:00:51.390+0000] {process_utils.py:190} INFO - Requirement already satisfied: cloudpickle<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.0.0)
[2024-04-23T00:00:51.391+0000] {process_utils.py:190} INFO - Requirement already satisfied: entrypoints<1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (0.4)
[2024-04-23T00:00:51.391+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitpython<4,>=2.1.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.1.43)
[2024-04-23T00:00:51.391+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyyaml<7,>=5.1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (6.0.1)
[2024-04-23T00:00:51.392+0000] {process_utils.py:190} INFO - Requirement already satisfied: protobuf<5,>=3.12.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (4.25.2)
[2024-04-23T00:00:51.393+0000] {process_utils.py:190} INFO - Requirement already satisfied: pytz<2024 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (2023.3.post1)
[2024-04-23T00:00:51.393+0000] {process_utils.py:190} INFO - Requirement already satisfied: requests<3,>=2.17.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (2.31.0)
[2024-04-23T00:00:51.394+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-metadata!=4.7.0,<8,>=3.7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (6.11.0)
[2024-04-23T00:00:51.394+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlparse<1,>=0.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (0.4.4)
[2024-04-23T00:00:51.394+0000] {process_utils.py:190} INFO - Requirement already satisfied: alembic!=1.10.0,<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.13.1)
[2024-04-23T00:00:51.395+0000] {process_utils.py:190} INFO - Requirement already satisfied: docker<8,>=4.0.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (7.0.0)
[2024-04-23T00:00:51.395+0000] {process_utils.py:190} INFO - Requirement already satisfied: Flask<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (2.2.5)
[2024-04-23T00:00:51.396+0000] {process_utils.py:190} INFO - Requirement already satisfied: scipy<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.10.1)
[2024-04-23T00:00:51.397+0000] {process_utils.py:190} INFO - Requirement already satisfied: querystring-parser<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.2.4)
[2024-04-23T00:00:51.397+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlalchemy<3,>=1.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.4.51)
[2024-04-23T00:00:51.398+0000] {process_utils.py:190} INFO - Requirement already satisfied: markdown<4,>=3.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.5.2)
[2024-04-23T00:00:51.398+0000] {process_utils.py:190} INFO - Requirement already satisfied: matplotlib<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.7.5)
[2024-04-23T00:00:51.399+0000] {process_utils.py:190} INFO - Requirement already satisfied: gunicorn<22 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (21.2.0)
[2024-04-23T00:00:51.400+0000] {process_utils.py:190} INFO - Requirement already satisfied: Jinja2<4,>=2.11 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.1.3)
[2024-04-23T00:00:51.423+0000] {process_utils.py:190} INFO - Requirement already satisfied: joblib>=1.1.1 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 3)) (1.4.0)
[2024-04-23T00:00:51.424+0000] {process_utils.py:190} INFO - Requirement already satisfied: threadpoolctl>=2.0.0 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 3)) (3.4.0)
[2024-04-23T00:00:51.429+0000] {process_utils.py:190} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.3.0)
[2024-04-23T00:00:51.430+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-resources in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (5.13.0)
[2024-04-23T00:00:51.434+0000] {process_utils.py:190} INFO - Requirement already satisfied: jmespath<2.0.0,>=0.7.1 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (0.10.0)
[2024-04-23T00:00:51.434+0000] {process_utils.py:190} INFO - Requirement already satisfied: s3transfer<0.11.0,>=0.10.0 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (0.10.1)
[2024-04-23T00:00:51.439+0000] {process_utils.py:190} INFO - Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (2.8.2)
[2024-04-23T00:00:51.440+0000] {process_utils.py:190} INFO - Requirement already satisfied: urllib3<1.27,>=1.25.4 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (1.26.18)
[2024-04-23T00:00:51.453+0000] {process_utils.py:190} INFO - Requirement already satisfied: Werkzeug>=2.2.2 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (2.2.3)
[2024-04-23T00:00:51.454+0000] {process_utils.py:190} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (2.1.2)
[2024-04-23T00:00:51.468+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitdb<5,>=4.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (4.0.11)
[2024-04-23T00:00:51.500+0000] {process_utils.py:190} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.8/site-packages (from importlib-metadata!=4.7.0,<8,>=3.7.0->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.17.0)
[2024-04-23T00:00:51.503+0000] {process_utils.py:190} INFO - Requirement already satisfied: MarkupSafe>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Jinja2<4,>=2.11->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (2.1.3)
[2024-04-23T00:00:51.519+0000] {process_utils.py:190} INFO - Requirement already satisfied: contourpy>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.1.1)
[2024-04-23T00:00:51.520+0000] {process_utils.py:190} INFO - Requirement already satisfied: cycler>=0.10 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (0.12.1)
[2024-04-23T00:00:51.520+0000] {process_utils.py:190} INFO - Requirement already satisfied: fonttools>=4.22.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (4.51.0)
[2024-04-23T00:00:51.520+0000] {process_utils.py:190} INFO - Requirement already satisfied: kiwisolver>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.4.5)
[2024-04-23T00:00:51.521+0000] {process_utils.py:190} INFO - Requirement already satisfied: pillow>=6.2.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (10.3.0)
[2024-04-23T00:00:51.522+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyparsing>=2.3.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.1.1)
[2024-04-23T00:00:51.582+0000] {process_utils.py:190} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.8/site-packages (from pandas<2.1.0,>=1.2.0->awswrangler==3.6.0->-r /tmp/venvm8jc_3g0/requirements.txt (line 1)) (2023.4)
[2024-04-23T00:00:51.594+0000] {process_utils.py:190} INFO - Requirement already satisfied: six in /home/***/.local/lib/python3.8/site-packages (from querystring-parser<2->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (1.16.0)
[2024-04-23T00:00:51.602+0000] {process_utils.py:190} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.3.2)
[2024-04-23T00:00:51.602+0000] {process_utils.py:190} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.6)
[2024-04-23T00:00:51.603+0000] {process_utils.py:190} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (2023.11.17)
[2024-04-23T00:00:51.664+0000] {process_utils.py:190} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from sqlalchemy<3,>=1.4.0->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (3.0.3)
[2024-04-23T00:00:51.731+0000] {process_utils.py:190} INFO - Requirement already satisfied: smmap<6,>=3.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venvm8jc_3g0/requirements.txt (line 2)) (5.0.1)
[2024-04-23T00:00:51.846+0000] {process_utils.py:190} INFO - Using cached awswrangler-3.6.0-py3-none-any.whl (403 kB)
[2024-04-23T00:00:51.848+0000] {process_utils.py:190} INFO - Using cached mlflow-2.10.2-py3-none-any.whl (19.5 MB)
[2024-04-23T00:00:53.659+0000] {process_utils.py:190} INFO - Installing collected packages: mlflow, awswrangler
[2024-04-23T00:00:53.659+0000] {process_utils.py:190} INFO -   Attempting uninstall: mlflow
[2024-04-23T00:00:53.661+0000] {process_utils.py:190} INFO -     Found existing installation: mlflow 2.12.1
[2024-04-23T00:00:53.661+0000] {process_utils.py:190} INFO -     Not uninstalling mlflow at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venvm8jc_3g0
[2024-04-23T00:00:53.661+0000] {process_utils.py:190} INFO -     Can't uninstall 'mlflow'. No files were found to uninstall.
[2024-04-23T00:00:54.941+0000] {process_utils.py:190} INFO - Successfully installed awswrangler-3.6.0 mlflow-2.10.2
[2024-04-23T00:00:55.311+0000] {process_utils.py:190} INFO - 
[2024-04-23T00:00:55.311+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.3.1 -> 24.0
[2024-04-23T00:00:55.311+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-04-23T00:00:55.437+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvm8jc_3g0/bin/python /tmp/venv-call9rqivrd5/script.py /tmp/venv-call9rqivrd5/script.in /tmp/venv-call9rqivrd5/script.out /tmp/venv-call9rqivrd5/string_args.txt /tmp/venv-call9rqivrd5/termination.log
[2024-04-23T00:00:55.443+0000] {process_utils.py:186} INFO - Output:
[2024-04-23T00:01:03.017+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.016+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-23T00:01:03.058+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.058+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-23T00:01:03.091+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.091+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-23T00:01:03.222+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.222+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-23T00:01:03.254+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.254+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-23T00:01:03.286+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.286+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-23T00:01:03.470+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.469+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-23T00:01:03.506+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.506+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-23T00:01:03.523+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.523+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-04-23T00:01:03.561+0000] {process_utils.py:190} INFO - [[34m2024-04-23T00:01:03.561+0000[0m] {[34mconfigprovider.py:[0m957} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-04-23T00:01:04.071+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-04-23T00:01:04.080+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_etl_heart_data, task_id=normalize_numerical_features, execution_date=20240422T000000, start_date=20240423T000048, end_date=20240423T000104
[2024-04-23T00:01:04.108+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T00:01:04.119+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
